murdock/job.py

```diff
@@ -235,8 +242,12 @@ async def extend_job_output(self, line):
                 json.dumps({"cmd": "output", "uid": self.uid, "line": line})
             )
 
+    def logger(self):
```

Comment:
If this really used ? Could be decorated with `@property` I think. Then all `self._logger` could be replaced with `self.logger`.

Reply:
There are a few cases in `murdock.py` where it is logging information about a job, so there it is easier to construct a new [bound logger](https://www.structlog.org/en/stable/bound-loggers.html) based on the job logger to include the job's context in the log line.

I agree that this could be cleaner, let me see what I can come up with.

Reply:
I rewrote this a bit, removing this function and adding a `logging_context` property here to expose the logging context that can be directly used with `LOGGER.bind` calls outside the `job` class